mlos_bench/mlos_bench/environments/local/local_env.py

```diff
@@ -187,6 +192,30 @@ def run(self) -> Tuple[Status, Optional[dict]]:
         _LOG.info("Local run complete: %s ::\n%s", self, data_dict)
         return (Status.SUCCEEDED, data_dict) if data_dict else (Status.FAILED, None)
 
+    def status(self) -> Tuple[Status, List[Tuple[datetime, str, Any]]]:
```

Comment:
We should have some documentation in a `README.md` somewhere that explains how this expected to work.  For instance:

```md
## Environment

Generally something that's tunable ...

### Lifecycle

1. `setup`
2. `run`
3. Async `status` polling during `run` phase.
    - Environments can optionally provide a `status` command in their config that can be executed to periodically generate telemetry info in csv format into a file in the shared storage area to be consumed asynchornously, or else return it on `stdout` to be consumed directly.
    - Data will then be stored in the DB for later analysis.
5. Results reporting/processing.
6. `teardown`
```

Reply:
Obviously a lot more to be filled in here.

Reply:
yes, I'll do that with the async version. rn I just want a working implementation of `.status()` and `.update_telemetry()` so I have something to use for debugging when working on the async stuff

Reply:
That's understandable.  I guess it'd help me while reviewing PRs to see what the overall design goal is, even if it's incomplete for now.